test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -74,7 +74,7 @@ public void Muxer_Exec_activation_of_Build_Output_Portable_DLL_with_DepsJson_Loc
             dotnet.Exec("exec", "--runtimeconfig", runtimeConfig, appDll)
                 .CaptureStdErr()
                 .CaptureStdOut()
-                .Execute()
+                .Execute(true)
```

Comment:
Also, that should be equivalent to `.Should().Fail() `used in Core-Setup and if so, that does not help mark the test as passed for the negative tests.